sdk/storage/Azure.Storage.Common/src/Shared/UriExtensions.cs

```diff
@@ -107,5 +108,13 @@ public static string GetAccountNameFromDomain(this Uri uri, string serviceSubDom
            !string.IsNullOrEmpty(uri.Host) &&
             uri.Host.IndexOf(".", StringComparison.InvariantCulture) >= 0 &&
             IPAddress.TryParse(uri.Host, out _);
+
+        internal static void AddQueryParam(this StringBuilder sb, string key, string value)
```

Comment:
Add an XML comment, maybe change the name to `AppendQueryParameter`, and put the `=>` on the same line as the method.